csharp/src/Google.Protobuf.Test/CodedInputStreamTest.cs

```diff
@@ -926,6 +926,39 @@ public void TestParseMessagesOver2G()
             }
         }
 
+        [Test]

```

Comment:
I'm not sure these test really belong under CodedInputStreamTest.  There's probably a better place where to put them?

Reply:
Also, by adding only a simple test like this you will only get a very basic testing. 

If you added a ReadOnlySpan codepath here: https://github.com/protocolbuffers/protobuf/blob/5e84a6169cf0f9716c9285c95c860bcb355dbdc1/csharp/src/Google.Protobuf.Test/MessageParsingHelpers.cs#L53 you'd get a much better coverage and then the Span-based overloads would automatically get a lot of test excercising them.

Reply:
Done!